content/zh/docs/reference/kubernetes-api/cluster-resources/namespace-v1.md

```diff
@@ -0,0 +1,807 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Namespace"
+content_type: "api_reference"
+description: "Namespace 为 NAMES 提供了一个作用域。"
+title: "Namespace"
+weight: 2
+auto_generated: true
+---
```

Comment:
Please add the original metadata to the comment

Ref: https://kubernetes.io/zh/docs/contribute/localization_zh/#frontmatter-%E7%9A%84%E5%A4%84%E7%90%86